computeResource.go

```diff
@@ -152,6 +152,13 @@ type computerResourceResponse struct {
 	Data ComputeResource `json:"data"`
 }
 
+func (s *ComputeResourcesService) List(ctx context.Context) ([]ComputeResource, error) {
+	resp := struct {
+		Data []ComputeResource `json:"data"`
+	}{}
+	return resp.Data, s.client.list(ctx, "compute_resources", &resp)
```

Comment:
I think we should return an paginated response here like we do in other methods.